experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/primitives.spec.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { SchemaBuilder } from "../../../feature-libraries";
+import { leaf } from "../../../domains";
+import { createTreeView, pretty } from "./utils";
+
+const _ = new SchemaBuilder({ scope: "test", libraries: [leaf.library] });
+const schema = _.toDocumentSchema(_.optional(leaf.all));
+
+const testCases = [
+	undefined,
+
+	// TODO: null,
+
+	true,
+	false,
+
+	-Infinity,
+	-Number.MAX_VALUE,
+	-Number.MIN_SAFE_INTEGER,
```

Comment:
I don't think you meant to negate this one ^.^ 

Also, I didn't know that `Infinity` was in the global scope. I thought we had to type `Number.PositiveInfinity` all this time. That makes infinity way nicer. Neat!

Reply:
I also just learned that `MIN_VALUE` is the closest number to _zero_, not the most negative number. `MIN_SAFE_INTEGER` is of course a very negative number. Not confusing at all.

Reply:
Haha.  Yeah, that was a mistake.  "One of these MIN_ constants is not like the others..."